src/Model/Scope/Condition.php

```diff
@@ -158,6 +167,11 @@ protected function onChangeModel(): void
         }
     }
 
+    protected function isTraversing()
```

Comment:
better not - originally we tested for is_string also for other things, then for `/` presence and then explode with `/` - this is more logical and fluent